### DISCUSSION TITLE
Unable to click "Select this school" button

### DISCUSSION THREAD

**Discussion by azarabishov** [community/community#116795] [GitHub Education]
### Hi everyone,

My application for "github education" has been approved. But at the last stage, I cannot click "select this school" in the section to select the school. I click but nothing happen.
![image](https://github.com/community/community/assets/165114618/0e8d9072-b64c-40b8-bf52-fd54d18680ec)

**Comment by nihilisticzebra** [comment]
> ### Hi everyone,
> 
> My application for "github education" has been approved. But at the last stage, I cannot click "select this school" in the section to select the school. I click but nothing happen.
> 
> ![image](https://github.com/community/community/assets/165114618/0e8d9072-b64c-40b8-bf52-fd54d18680ec)
> 
> 

Same here

**Reply by azarabishov** [reply]
@nihilisticzebra Continue button will be enabled after your student packs will be available. This can take 2 3 days after approve.
